website/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "gatsby-docs-starter",
```

Comment:
can we update the properties in this?

Reply:
Should we? The website is effectively its own project, and understanding where it came from to make it easier to chase down seems useful.

Reply:
Yep, but it is our project. So we can change the name to `react-beautiful-dnd-docs` 👍